makefile

```diff
@@ -1,7 +1,16 @@
 install-dep:
 	pip install -r requirements.txt
-app-run:install-dep
-	flask run
+
+run:install-dep
+	flask -A student.py run --host=${HOSTPORT} --port=${APP_PORT}
+
+run-app:run
+
+#Push the docker image
+docker_push:
+	docker tag studentapi:$(IMAGE_TAG) ${USERNAME}/studentapi:1.0.0
+	docker push ${USERNAME}/studentapi:1.0.0
```

Comment:
This should be `docker push ${USERNAME}/studentapi:${IMAGE_TAG}`

Reply:
Tagging a image is required to push into docker hub.